site/package-lock.json

```diff
@@ -1576,7 +1585,7 @@
     },
     "bl": {
       "version": "1.2.2",
-      "resolved": "https://registry.npmjs.org/bl/-/bl-1.2.2.tgz",
+      "resolved": "http://registry.npmjs.org/bl/-/bl-1.2.2.tgz",
```

Comment:
I'm not sure why this was downgraded to HTTP, probably NPM goofing up

Reply:
Looks like a registry issue: https://npm.community/t/some-packages-have-dist-tarball-as-http-and-not-https/285/18
Gona sed it all with 
```sh
sed -i -e 's/http:\/\//https:\/\//g' package-lock.json
```